README.md

```diff
@@ -31,10 +31,15 @@ This project tests all builds against each supported version of Fedora, the late
 
 ## Out of scope
 
-* Signing and pushing images to various image storages.
-  See [Skopeo](https://github.com/containers/skopeo/).
-* Container Runtimes daemons for working with the Kubernetes CRI interface.
-  See [CRI-O](https://github.com/kubernetes-sigs/cri-o).
+* Specializing in signing and pushing images to various storage backends.
+  See [Skopeo](https://github.com/containers/skopeo/) for those tasks.
+* Container runtimes daemons for working with the Kubernetes CRI interface.
+  [CRI-O](https://github.com/kubernetes-sigs/cri-o) specializes in that.
+* Supporting `docker-compose`.  We believe that Kubernetes is the defacto
+  standard for container orchestration.  Hence, Podman allows the creation of
+  Pods from a Kubernetes YAML file. See
```

Comment:
This doesn't make sense as a term for `out of scope`. `docker-compose` is not the same as Docker Swarm. Essentially, `docker-compose` is a way to spin up a pod on your machine using your single container engine instance.

Making someone deploy Kubernetes for that is madness. However, it is nice that podman can generate Kubernetes YAML for translation to a Kubernetes system, it makes no sense to not have the facility built in so that people _can_ develop these things to eventually host in an OpenShift/Kubernetes system.

Reply:
docker compose is not part of Docker it is a separate project.  And it is Huge. So we currently do not have the man power to do everything that is involved in doing this. If someone wants to start a project called podman-compose, then we would support them.

We don't want to force anyone to do kubernetes.  We are just suggesting alternatives and looking at the problem from a different point of view.

Reply:
> This doesn't make sense as a term for `out of scope`. `docker-compose` is not the same as Docker Swarm. Essentially, `docker-compose` is a way to spin up a pod on your machine using your single container engine instance.

Right. You can use `podman play kube` in a similar fashion but with Kubernetes yaml.

> Making someone deploy Kubernetes for that is madness. However, it is nice that podman can generate Kubernetes YAML for translation to a Kubernetes system, it makes no sense to not have the facility built in so that people _can_ develop these things to eventually host in an OpenShift/Kubernetes system.

I suggest cooling down a bit.

Noone is forcing users into Kubernetes.  `podman play kube` is meant for the exact use-case you describe but it requires a Kubernetes yaml instead of Compose yaml.

You were referring to `podman generate kube` which generates this yaml file based on a set of containers/pods and allows migrating to Kubernetes, if desired.

However, the text must be corrected as it was meant to refer to podman-play-kube.

Reply:
> However, the text must be corrected as it was meant to refer to podman-play-kube.

Actually, it's fine in this regard: "Hence, Podman allows the creation of Pods from a Kubernetes YAML file."

Not sure if we should avoid mentioning "orchestrating" and rephrase it.